src/test/java/io/confluent/examples/streams/AggregateTest.java

```diff
@@ -75,11 +75,15 @@ public void shouldAggregate() {
 
     try (final TopologyTestDriver testDriver = new TopologyTestDriver(builder.build(), streamsConfiguration)) {
       // Step 2: Write the input
-      IntegrationTestUtils.produceValuesSynchronously(inputTopic, inputValues, testDriver, new StringSerializer());
+      testDriver.createInputTopic(inputTopic,
+                                  new IntegrationTestUtils.NothingSerde<>(),
+                                  new StringSerializer())
+        .pipeValueList(inputValues);
```

Comment:
Tentatively, I think it might set a better example for users if we establish a pattern like:
```
try (testDriver = new TopologyTestDriver(...)) {
  final TestInputTopic<...> inputTopic = testDriver.createInputTopic(...);
  final TestOutputTopic<...> outputTopic = testDriver.createOutputTopic(...);

  inputTopic.pipeValueList(inputValues);
  assertThat(outputTopic.readKeyValuesToMap()).isEqualTo(expectedOutput);
}
```

In other words, I feel it leads to more readable tests if we declare the inputs and outputs right up front, and then push data through and assert the results later on.